src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -7437,7 +7437,7 @@ components:
           $ref: '#/components/schemas/SnapshotAccessRequestStatus'
         createdDate:
           type: string
-        updatedDate:
+        statusUpdatedDate:
```

Comment:
Do we have UI's that use this yet? This is technically a breaking change, so we should make sure we aren't breaking things/have a UI update to fix it staged.

Reply:
good point! I just checked both terra-ui and jade-data-repo ui and it looks like this field isn't used anywhere